Makefile

```diff
@@ -209,9 +211,9 @@ verify: .init .generate_files verify-client-gen
 	    | grep -v v1alpha1/defaults.go); \
 	  do \
 	    golint --set_exit_status $$i; \
-	    go vet $$i; \
 	  done'
 	@#
+	$(DOCKER_CMD) go vet $(NON_VENDOR_DIRS)
```

Comment:
would using $NON_VENDOR_DIRS on the "golint" step work too? Does it result in checking the same set of files?